src/wmnds/components/content-card/_content-card.njk

```diff
@@ -0,0 +1,24 @@
+{% macro wmndsContentCard(params) %}
+{# Set #}
+{% set variantModifier = " " + params.variant if params.variant else "" %}
+{% set yellowModifier = " wmnds-content-card--yellow" if params.yellow else "" %}
+{# Opening & closing tags #}
+{% set openingTag = "a href=\"#\" target=\"_self\"" if params.clickable else "div" %}
+{% set closingTag = "a" if params.clickable else "div" %}
+{# HTML content #}
+{% set html = params.html if params.html else "
+    <div class=\"wmnds-p-lg wmnds-m-lg\">
```

Comment:
```suggestion
    <div class=\"wmnds-p-sm\">
```
I know you've used large margin/padding so that the samples look better on the card component page. 
But I think it'll be better if you used wmnds-p-sm, as it is used throughout all the card patterns.

Maybe put some dummy data inside this section to make up for the lost wmnds-p-lg and wmnds-m-lg.